### DISCUSSION TITLE
Custom darkMode class

### DISCUSSION THREAD

**Discussion by estruyf** [tailwindlabs/tailwindcss#5294] [Help]
Is there a way to use a custom dark mode class? I'm integrating withing another system and they provide a custom class to specify if they are in light or dark mode. Instead of using my own handler, it would be great if I can reuse the class name from the app instead of `dark`. Is this possible?

**Comment by aggreggator** [comment]
Currently it's hardcoded:
https://github.com/tailwindlabs/tailwindcss/blob/0413b84c12e44950b84b144fafaffb5266f371b8/src/jit/corePlugins.js#L280-L282
You can add a prefix, but this would apply to all selectors.
